apps/web/.env

```diff
@@ -2,3 +2,6 @@ NEXT_PUBLIC_MM_API_URL=https://linked-pool.pancakeswap.com/quote-service
 NEXT_PUBLIC_QUOTING_API=https://swap-quoting.pancakeswap.com/quoting-service
 MOONPAY_API="https://api.moonpay.com"
 MOONPAY_PUBLISHABLE_KEY="pk_live_Ch5fat39X8NvMZwih2k7hK4sDrKanSPz"
+NEXT_PUBLIC_PROJECT_ID="ae5413feaf0cdaee02910dc807e03203"
```

Comment:
Can we rename it to `NEXT_PUBLIC_NOTIFICATION_PROJECT_ID`?

Reply:
sure. renamning in next commit